config.go

```diff
@@ -1378,3 +1387,27 @@ func normalizeNetwork(network string) string {
 
 	return network
 }
+
+// We need to detect the chain by seeing if one is specified in the configuration file.
```

Comment:
Nit: function comment should be of the form `getBitcoindRPCCredentialsFromCookie ...` that describes what the function does at a glance, like all the others.